test/rendering/commands/update-screenshots.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -x
+
+# To stop execution instantly as a query exits while having a non-zero status
+# To know the error location in the running code
+set -e
+
+# Create and start the container with the name *sn-tale-playwright*
+# using the Docker host network stack and binding your current directory
+# in the background in a “detached” mod and remove the container once it exits/stops
+# The -it instructs Docker to allocate a pseudo-TTY connected to the container’s stdin;
+# creating an interactive bash shell in the container
+docker run -d --name sn-tale-playwright  --rm --network host -v $(pwd):/work/ -w /work/ -it mcr.microsoft.com/playwright:v1.19.2-focal
+
+# The actual generate/update the reference screenshots is running from the machine running this shell script
+docker exec sn-tale-playwright /bin/sh -c "yarn test:rendering --update-snapshots"
```

Comment:
Can't you have update snapshots as an option for the run-rendering-test.sh, since they are identical otherwise?

Reply:
i also think about that and i like the different file names for each task

Reply:
That is a lot of duplicate code though, you would have to change both if we update the flow

Reply:
It is fine though, as long as you keep them synced